ale/drivers/__init__.py

```diff
@@ -32,7 +33,8 @@
 __driver_modules__ = [importlib.import_module('.'+m, package='ale.drivers') for m in __all__]
 
 __formatters__ = {'usgscsm': to_usgscsm,
-                  'isis': to_isis}
+                  'isis': to_isis,
+                  'ale' : to_isd}
```

Comment:
Seems like a way to get functionality for instruments without full support for both ISIS and CSM formatters is to simply make the singular ISD a new formatter and slowly start phasing out the other two.